.circleci/config.yml

```diff
@@ -1493,6 +1503,7 @@ workflows:
             # - kinesisvideo-archivedmedia
             # - kinesisvideo
             - mobile-client
+            - textract
```

Comment:
This same line should be added to the release_sdk workflow

Reply:
Added